sharepoint/sharepoint-ps/sharepoint-online/Set-SPORestrictedSiteCreation.md

```diff
@@ -0,0 +1,178 @@
+---
+external help file: sharepointonline.xml
+Module Name: Microsoft.Online.SharePoint.PowerShell
+online version: https://learn.microsoft.com/powershell/module/sharepoint-online/set-sporestrictedsitecreation
+applicable: SharePoint Online
+title: Set-SPORestrictedSiteCreation
+schema: 2.0.0
+author: vgaddam-pm
+ms.author: vgaddam
+ms.reviewer: jmcdowe
+---
+
+# Set-SPORestrictedSiteCreation
+
+## SYNOPSIS
+
+Sets or updates one or more group configurations for restricting site creation.
+
+## SYNTAX
+
+### ParamSet1
+
+```powershell
+Set-SPORestrictedSiteCreation
+   [-Enabled <Boolean>]
+   [-Mode <RestrictedSiteCreationMode>]
+   [-SiteType <RestrictedSiteCreationSiteType>]
+   [-RestrictedSiteCreationGroups <String>]
+```
+
+## DESCRIPTION
+
+The Set-SPORestrictedSiteCreation cmdlet sets or updates the configuration or setting for the Restricted Site Creation feature.
+
+You must be a SharePoint Online administrator to run the cmdlet.
```

Comment:
remove this line -- we only include it if it requires a different permission other than SharePoint admin, since by definition cmdlets can only be executed by SharePoint admin